### PR TITLE
remote debugging changes (testing)

### PR DESCRIPTION
**Pull request by tmadanagopal**

### CODE REVIEW

**Comment by tmadanagopal** [comment]
need to address Nikita Comments on inspector running check

File: examples/pxScene2d/external/node-v8.15.1_mods.patch [hunk 1]
```diff
@@ -1,324 +1,5 @@
-diff --git a/examples/pxScene2d/external/libnode-v8.15.1/common.gypi b/examples/pxScene2d/external/libnode-v8.15.1/common.gypi
```
Comment (npoltorapavlo) [inline examples/pxScene2d/external/node-v8.15.1_mods.patch]: Is there any reason why you are reverting so many changes from node-v8.15.1_mods.patch?

File: examples/pxScene2d/external/node-v8.15.1_mods.patch [hunk 2]
```diff
@@ -1,324 +1,5 @@
...
-+/*MODIFIED CODE END*/
-+
-   debug('connect: find host', host);
-   debug('connect: dns options', dnsopts);
-   self._host = host;
 diff --git a/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc b/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
-index 3d791f0..d572a6d 100644
+index 3d791f0..0d810a9 100644
 --- a/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
 +++ b/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
```
Comment (npoltorapavlo) [inline examples/pxScene2d/external/node-v8.15.1_mods.patch]: You should not modify files in libnode-v8.15.1. Add changes to node-v8.15.1_mods.patch.

File: examples/pxScene2d/external/node-v8.15.1_mods.patch (hunk 1 above)
Comment (tmadanagopal) [reply examples/pxScene2d/external/node-v8.15.1_mods.patch]: I will re-check and re-raise. No code in current patch need to to be removed.

File: examples/pxScene2d/external/node-v8.15.1_mods.patch (hunk 2 above)
Comment (tmadanagopal) [reply examples/pxScene2d/external/node-v8.15.1_mods.patch]: fine

**Pull request closed without merging**